kolibri_sentry_plugin/assets/src/module.js

```diff
@@ -1,46 +1,40 @@
-import * as Sentry from '@sentry/browser';
-import * as SentryIntegrations from '@sentry/integrations';
+import * as Sentry from '@sentry/vue';
 import kolibri from 'kolibri';
-import plugin_data from 'plugin_data';
-import Vue from 'kolibri.lib.vue';
-import logger from 'kolibri.lib.logging';
-import store from 'kolibri.coreVue.vuex.store';
-import { currentLanguage } from 'kolibri.utils.i18n';
+import plugin_data from 'kolibri-plugin-data';
+import Vue from 'vue';
+import logger from 'kolibri-logging';
+import store from 'kolibri/store';
+import { currentLanguage } from 'kolibri/utils/i18n';
 
 const logging = logger.getLogger(__filename);
```

Comment:
Is this logging constant still relevant since the error and warning logging have been removed below?

Reply:
Probably not no, and as I failed to add pre-commit, linting did not warn me it was unused!